lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,12 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
```

Comment:
I wonder if the writer is necessary and whether the config value should be the only way to initialize this?